genstudio-mlr-claims-app/src/genstudiopem/web-src/src/components/RightPanel.tsx

```diff
@@ -13,7 +13,7 @@ governing permissions and limitations under the License.
 import React, { Key, useEffect, useState } from 'react';
 import { attach } from "@adobe/uix-guest";
 import { extensionId, TEST_CLAIMS } from "../Constants";
-import { View, Provider, defaultTheme, Button, ComboBox, Item, Flex, Divider, Picker, Text } from '@adobe/react-spectrum';
+import { View, Provider, defaultTheme, Button, ComboBox, Item, Flex, Divider, Picker, Text, Heading, ProgressCircle } from '@adobe/react-spectrum';
```

Comment:
had to use the progress circle approach because it is a smaller circle than the spinner